pydocstringformatter/formatting/formatter.py

```diff
@@ -12,19 +12,26 @@ class BeginningQuotesFormatter(StringFormatter):
     """Fix the position of the opening quotes."""
 
     name = "beginning-quotes"
+    potential_single_line = re.compile(r"""['"]{1,3}\n\s*.+\n\s*['"]{1,3}""")
```

Comment:
```suggestion
    potential_single_line = re.compile(r"""
        ['"]{1,3} # 3 opening quotes
        \n\s*.+\n\s* # ...
        ['"]{1,3} # 3 ending quote
    """)
```
Would something like this be helpful ? It's hard to maintain regex but maybe this one is simple enough.

Reply:
```suggestion
    potential_single_line = re.compile(r"""
        ['"]{1,3}         # 3 opening quotes
        \n\s*.+           # A line with any length of characters
        \n\s*              # A line with only whitespace
        ['"]{1,3}         # 3 ending quote
    """)
```